src/operator/src/request.rs

```diff
@@ -90,6 +90,39 @@ impl Requester {
         .await
     }
 
+    /// Handle the request to build index for table.
+    pub async fn handle_table_build_index(
+        &self,
+        request: BuildIndexTableRequest,
+        ctx: QueryContextRef,
+    ) -> Result<AffectedRows> {
+        let partitions = self
+            .get_table_partitions(
+                &request.catalog_name,
+                &request.schema_name,
+                &request.table_name,
+            )
+            .await?;
+
+        let requests = partitions
+            .into_iter()
+            .map(|partition| {
+                RegionRequestBody::BuildIndex(BuildIndexRequest {
+                    region_id: partition.id.into(),
+                })
+            })
+            .collect();
+
+        info!("Handle table manual build index request: {:?}", request);
```

Comment:
Log the request at debug level instead of info, and log only the table name at info level.